### DISCUSSION TITLE
Star Button Drop-down Menu has z-index issue when clicking "Create List"

### DISCUSSION THREAD

**Discussion by benlmyers** [community/community#8711] [Other Feature Feedback, Questions, & Ideas]
Another great feature introduced by the GitHub team! Thank you for keeping the platform fresh with new features.

The drop-down menu that appears when pressing the disclosure button (▼) next to the Star count has a layering issue when the mouse is down on the "**Create List**" button.

The screenshot below displays the issue. Please note that the "**Create List**" button is actively being mouse-pressed down while the screenshot is taken.

<img width="1680" alt="Screen Shot 2021-12-11 at 7 52 05 AM" src="https://user-images.githubusercontent.com/41796709/145682995-e46f0478-196a-4009-aaf3-b77876ced40e.png">

**Comment by brianlovin** [comment]
Thanks for the heads up! The team is looking into a fix for this that will hopefully be out soon.

**Reply by benlmyers** [reply]
Sounds great! Thank you @brianlovin. Best of luck with Lists!
